### DISCUSSION TITLE
Control-module request values (8FB range)

### DISCUSSION THREAD

**Discussion by batal131** [jilleb/MQB-FPA#47] [Exploration and research]
Hey all, 

Do anyone know how to properly work with this part of dataset? 
(Control-module request values (8FB range))

I've successfully ungrouped some controls.  But during that I had to reorder them (controls)

Not sure what I did wrong, but seems like now when I have "normal" for ESC in profile, it works as ESC SPORT. For me it looks like I need to change Control-module request values (8FB range) table. But I don't know what this values are and what is columns for it.

In attachment you can find my dataset (maybe someone have have time to take a loo
[NC_MEMORY_ENGINE_ALL_V4.zip](https://github.com/jilleb/MQB-FPA/files/7539367/NC_MEMORY_ENGINE_ALL_V4.zip)
k)

**Comment by vivacuba** [comment]
Try:
16 50 00 00 00 00 00 00
00 02 04 00 00 00 00 00 
00 50 04 00 00 10 00 00

**Reply by MilackB** [reply]
Hi
"8FB" means  ROW: 08F0h　Column B
You can find  "10".  Hex10 means Dec16  　
Name is ”ubyte control_request_value_0[30]”
 

> Hey all,
> 
> Do anyone know how to properly work with this part of dataset? (Control-module request values (8FB range))
> 
> I've successfully ungrouped some controls. But during that I had to reorder them (controls)
> 
> Not sure what I did wrong, but seems like now when I have "normal" for ESC in profile, it works as ESC SPORT. For me it looks like I need to change Control-module request values (8FB range) table. But I don't know what this values are and what is columns for it.
> 
> In attachment you can find my dataset (maybe someone have have time to take a loo [NC_MEMORY_ENGINE_ALL_V4.zip](https://github.com/jilleb/MQB-FPA/files/7539367/NC_MEMORY_ENGINE_ALL_V4.zip) k)
![pic](https://user-images.githubusercontent.com/96638842/147412388-d23ef11c-acc0-4390-bc1c-34c925511db3.jpg)
